deploy/README.md

```diff
@@ -0,0 +1,16 @@
+# Deploy Kamaji
```

Comment:
not sure the reason for this README

Reply:
Personally I'd like to have from projects that can be deployed, also brief explanation on this type of standard directories, at least to say "what you should expect from deploy tools you find here"

Reply:
Thanks got it